CHUI-77: Fix mandatory invoice address (#1339)
vtex/omnishipping: CHANGELOG.md

## [Unreleased]

### Fixed

> - Now the "go to payment" button, when the invoice is active, is only present when there's a valid invoice address.

Reviewer: ```suggestion
- "go to payment" button's presence only on valid invoice address.
```
